### DISCUSSION TITLE
Cannot Render bigint as signal value : Is it intended behavior ?

### DISCUSSION THREAD

**Discussion by coleea** [solidjs/solid#2117] [General]
Hi.

When I use bigint value for createSignal, It cannot be rendered directly : I should wrap it like this `{String(someValue())}` instead of `{someValue()}`.

Is it intended behavior or bug ? 

Thank you.

![1제목 없음](https://github.com/solidjs/solid/assets/7471674/855e5101-e65b-4389-9f56-732f27b31c34)

**Comment by petercockerell-brt** [comment]
I think its more a JSX issue. A JSX `Element` is `Node | ArrayElement | (string & {}) | number | boolean | null | undefined`, so anything other than those would need to be stringified.
